docs/code-standards-policy.rst

```diff
@@ -151,3 +206,87 @@ a dependency and pass a specific configuration to it:
 By including the configuration for the debops.apt_preferences_ role in your role's
 default variables you allow the user to change it through the Ansible inventory
 without the need to modify any of the involved roles or the playbook.
+
+.. _policy__ref_default_variable_documentation:
+
+Variable documentation
+~~~~~~~~~~~~~~~~~~~~~~
+
+For each role the `DebOps Documentation`_ is will include a page which
+documents the default variables. This page is generated from the role's
+:file:`defaults/main.yml` file with help of yaml2rst_. The entire comment of
+the defauls file is thereby interpreted as reStructuredText_ and then rendered
+via Sphinx_.
+
+Each variable comment is started with a ``.. envvar::`` reference anchor
+followed by the name of the variable. This construct allows any documentation
+page to reference the variable via ``:envvar:`varname``` which :program:`Sphinx`
+will translate into a link pointing to the variable description. Below the
+anchor a comment should describe the purpose of the variable, accepted values,
+side effects and so on. Within the comment all reStructuredText constructs
+supported by :program:`Sphinx` can be used. An example variable definition would
+eventually look like this:
+
+.. code-block:: yaml
+
+    # .. envvar:: ferm__flush [[[
+    #
+    # Should ferm-rules be flushed when :program:`ferm` is disabled? The default is true,
+    # but you may need set both :envvar:`ferm__enabled` and this to ``False`` if you are
+    # running in some container and are not allowed to change :command:`iptables`.
+    ferm__flush: '{{ ferm__enabled | bool }}'
+                                                                   # ]]]
+
```

Comment:
I think there is still an entire section "Documentation" missing. But I'd first like to get this ready for merging before opening another chapter.